core/src/test/java/io/substrait/type/proto/WindowFunctionRoundtripTest.java

```diff
@@ -1,71 +0,0 @@
-package io.substrait.type.proto;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import io.substrait.TestBase;
-import io.substrait.expression.Expression;
-import io.substrait.expression.ExpressionCreator;
-import io.substrait.expression.ImmutableExpression;
-import io.substrait.expression.ImmutableWindowBound;
-import io.substrait.extension.ExtensionCollector;
-import io.substrait.relation.ConsistentPartitionWindow;
-import io.substrait.relation.ImmutableConsistentPartitionWindow;
-import io.substrait.relation.ImmutableVirtualTableScan;
-import io.substrait.relation.ProtoRelConverter;
-import io.substrait.relation.RelProtoConverter;
-import io.substrait.relation.VirtualTableScan;
-import io.substrait.type.TypeCreator;
-import java.io.IOException;
-import java.math.BigDecimal;
-import java.util.Arrays;
-import java.util.Collections;
-import org.junit.jupiter.api.Test;
-
-public class WindowFunctionRoundtripTest extends TestBase {
```

Comment:
Renamed to `ConsistentPartitionWindowRelRoundtripTest` as we're testing the roundtripability of the rel as a whole.